modules/tf_cloudbuild_builder/variables.tf

```diff
@@ -109,3 +109,16 @@ variable "dockerfile_repo_type" {
     error_message = "Must be one of UNKNOWN, CLOUD_SOURCE_REPOSITORIES, GITHUB or BITBUCKET_SERVER."
   }
 }
+
+variable "enable_worker_pool" {
```

Comment:
Do we need this flag due to known before apply issues?

Reply:
yes, it is because of the known before apply issue.